src/lib/tabs/tab-group.ts

```diff
@@ -275,4 +279,42 @@ export class MdTabGroup extends _MdTabGroupMixinBase implements AfterContentInit
     this._tabBodyWrapperHeight = this._tabBodyWrapper.nativeElement.clientHeight;
     this._renderer.setStyle(this._tabBodyWrapper.nativeElement, 'height', '');
   }
+
+  /** Body content was swiped left/right */
+  _bodyContentSwiped(event: HammerInput): void {
+    if (this.selectedIndex === null) {
+      this.selectedIndex = 0;
+    }
+
+    if (event.direction === HammerDirection.Left || event.direction === HammerDirection.Right) {
+      let direction = event.direction;
+      if (this._dir.value === 'rtl') {
+        direction =
+          direction === HammerDirection.Left ? HammerDirection.Right : HammerDirection.Left;
+      }
+
+      if (this.selectedIndex !== 0 && direction === HammerDirection.Right) {
+        this._setActiveItemByIndex(this.selectedIndex - 1, -1);
+      } else if (this.selectedIndex < this._tabs.length && direction === HammerDirection.Left) {
+        this._setActiveItemByIndex(this.selectedIndex + 1, 1);
+      }
+    }
+  }
+
+  /**
+   * Sets the active item to the first enabled item starting at the index specified. If the
+   * item is disabled, it will move in the fallbackDelta direction until it either
+   * finds an enabled item or encounters the end of the list.
+   */
+  private _setActiveItemByIndex(index: number,
+                                fallbackDelta: number,
+                                items = this._tabs.toArray()): void {
```

Comment:
Why include the items array as a param? I don't see it passed in by anyone

Reply:
It was a pattern from another component that I used that @jelbourn referred me to. Do you think it should just be an variable defined in the function body?

Reply:
Can remove it if the param is never passed